service/src/worker_peers_updater.rs

```diff
@@ -35,19 +35,19 @@ pub trait UpdateWorkerPeers {
 	fn update_peers(&self) -> WorkerResult<()>;
 }
 
-pub struct GlobalPeerUpdater<Worker> {
+pub struct WorkerPeersUpdater<Worker> {
```

Comment:
Re-named from `GlobalPeerUpdater` to `WorkerPeersUpdater`, because  this updater itself is not global or has any global component per se.